frontend/src/PagesAdmin/RecruitmentFormAdminPage/RecruitmentFormAdminPage.tsx

```diff
@@ -67,133 +62,168 @@ export function RecruitmentFormAdminPage() {
+                      <Input type="datetime-local" {...field} />
+                    </FormControl>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+            </div>
+            <div className={styles.row}>
+              <FormField
+                control={form.control}
+                name="shown_application_deadline"
+                render={({ field }) => (
+                  <FormItem className={styles.item}>
+                    <FormLabel>{t(KEY.shown_application_deadline)}</FormLabel>
+                    <FormControl>
+                      <Input type="datetime-local" {...field} />
+                    </FormControl>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+              <FormField
+                control={form.control}
+                name="actual_application_deadline"
+                render={({ field }) => (
+                  <FormItem className={styles.item}>
+                    <FormLabel>{t(KEY.actual_application_deadline)}</FormLabel>
+                    <FormControl>
+                      <Input type="datetime-local" {...field} />
+                    </FormControl>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+            </div>
+            <div className={styles.row}>
+              <FormField
+                control={form.control}
+                name="reprioritization_deadline_for_applicant"
+                render={({ field }) => (
+                  <FormItem className={styles.item}>
+                    <FormLabel>{t(KEY.reprioritization_deadline_for_applicant)}</FormLabel>
+                    <FormControl>
+                      <Input type="datetime-local" {...field} />
+                    </FormControl>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+              <FormField
+                control={form.control}
+                name="reprioritization_deadline_for_groups"
+                render={({ field }) => (
+                  <FormItem className={styles.item}>
+                    <FormLabel>{t(KEY.reprioritization_deadline_for_groups)}</FormLabel>
+                    <FormControl>
+                      <Input type="datetime-local" {...field} />
+                    </FormControl>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+            </div>
+            <div className={styles.row}>
+              <FormField
+                control={form.control}
+                name="max_applications"
+                render={({ field }) => (
+                  <FormItem className={styles.item}>
+                    <FormLabel>{t(KEY.max_applications)}</FormLabel>
+                    <FormControl>
+                      <Input type="number" {...field} value={field.value ?? undefined} />
```

Comment:
Skal ikke være nødvendig å sette value her

```suggestion
                      <Input type="number" {...field} />
```

Reply:
Er gjort fordi Input komponenten ikke tok hensyn til null values. Vet ikke om dagens løsning er bedre, men men